provisioning/package.json

```diff
@@ -33,6 +33,7 @@
     "babel-eslint": "^10.1.0",
     "eslint": "^7.15.0",
     "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-prettier": "^8.1.0",
```

Comment:
what's our current lint/prettier situation? Do we follow a specific way to use prettier in combination with eslint in all projects?

Reply:
Adjusted the eslint to rules of frontend since both are javascript projects. Also changed eslint config file from eslintrc.yml to .eslintrc for consistency.

Reply:
Provisioning lint is already added in #794 , this commit can be removed